internal/cmd/egctl/translate.go

```diff
@@ -93,6 +96,9 @@ func newTranslateCommand() *cobra.Command {
   # Translate Gateway API Resources into All xDS Resources in YAML output,
   # also print the Gateway API Resources with updated status in the same output.
   egctl experimental translate --from gateway-api --to gateway-api,xds --type all --output yaml --file <input file>
+
+  # Translate Gateway API Resources into IR in YAML output,
```

Comment:
can you also update the user doc? https://gateway.envoyproxy.io/latest/user/egctl/#egctl-experimental-translate thx